docs/the_black_code_style/current_style.md

```diff
@@ -8,12 +8,14 @@ deliberately limited and rarely added. Previous formatting is taken into account
 little as possible, with rare exceptions like the magic trailing comma. The coding style
 used by _Black_ can be viewed as a strict subset of PEP 8.
 
-_Black_ reformats entire files in place. It doesn't reformat lines that end with
+_Black_ reformats entire files in place. It doesn't reformat lines that contain
 `# fmt: skip` or blocks that start with `# fmt: off` and end with `# fmt: on`.
-`# fmt: on/off` must be on the same level of indentation and in the same block, meaning
-no unindents beyond the initial indentation level between them. It also recognizes
-[YAPF](https://github.com/google/yapf)'s block comments to the same effect, as a
-courtesy for straddling code.
+`# fmt: skip` can be mixed with other pragmas/comments either with multiple comments
+(e.g. `# fmt: skip # pylint # noqa`) or as a semicolon separeted list (e.g.
+`# fmt: skip; pylint; noqa`). `# fmt: on/off` must be on the same level of indentation
```

Comment:
I didn't see requests for this semicolon separation in the issue. Do we really need it?

Reply:
I picked it up from [this comment](https://github.com/psf/black/issues/2213#issuecomment-1411226962). I thought it was good, as I personally don't like the double comment syntax, but I'm open to drop it if you think it's better that way.

Reply:
It's probably a good idea! Let's keep it.